lib/compact.js

```diff
@@ -393,14 +393,6 @@ api.compactIri = ({
 
   const inverseCtx = activeCtx.getInverse();
 
-  // if term is a keyword, it can only be compacted to a simple alias
-  if(_isKeyword(iri)) {
-    if(iri in inverseCtx) {
-      return inverseCtx[iri]['@none']['@type']['@none'];
-    }
-    return iri;
-  }
-
```

Comment:
Another option would be to leave this in place, and exit quickly if an `iri` is found, and otherwise continue. That would allow for more complicated behavior, while keeping the typical case fast.